trade-finance-manager-ui/server/controllers/login/loginNonSso/index.js

```diff
@@ -37,12 +37,12 @@ const postLogin = async (req, res) => {
   }
 
   if (loginErrors.length) {
-    return res.render('login.njk', {
+    res.render('login.njk', {
       errors: validationErrorHandler(loginErrors),
     });
   }
 
-  return res.redirect('/home');
+  res.redirect('/home');
```

Comment:
Why are the returns removed?

Reply:
Good spot - I dont think we technically need them (although it might be that we dont technically need them only on the last call of `res`, in which case thats inconsistent. reverted